projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -49,13 +49,6 @@ abstract class Publicize_Base {
 	 */
 	public $POST_MESS = '_wpas_mess';
 
-	/**
-	 * Post meta key for flagging when the post is a tweetstorm.
-	 *
-	 * @var string
-	 */
-	public $POST_TWEETSTORM = '_wpas_is_tweetstorm';
```

Comment:
Since we're not using this anymore, maybe we can remove it from the sync post meta too?

https://github.com/Automattic/jetpack/blob/1039bd3005c64e43089e38e7264866bef53d793e/projects/packages/sync/src/class-defaults.php#L743

Reply:
Good point

Reply:
I created a Phab patch for the wpcom change: D132176-code

So we have to merge that first once again?

Reply:
You can merge it now indeed. 👍